### PR TITLE
Add graduation announcement

### PR DESCRIPTION
**Pull request by lucperkins**

### CODE REVIEW

**Comment by lucperkins** [comment]
cc @estesp

**Comment by estesp** [comment]
Cool! I like it. The only problem is Travis CI would like to see a `Signed-off-by: ` on your commits :)

**Comment by lucperkins** [comment]
@estesp Hmmm. Both commits have a DCO sign-off.

**Comment by lucperkins** [comment]
@estesp Ah, I see now. There was a whitespace error in a commit. I forgot that *all* commits have to have that error addressed. I got bitten by this before. Should be okay now.

File: layouts/partials/home/graduation.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<section class="section has-background-white-bis graduation">
+  <div class="container">
+    <div class="columns is-vcentered">
+      <div class="column">
+        <p class="title is-size-3 is-size-4-mobile has-text-weight-light">
+          containerd has graduated from the <a href="https://cncf.io">CNCF</a>!
```
Comment (mikebrow) [inline layouts/partials/home/graduation.html]: /s/has graduated from CNCF/is now a CNCF graduated project/

IOW we are not "leaving".. :-)

File: layouts/partials/home/graduation.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        </p>
+
+        <hr />
+
+        <p class="is-size-5 is-size-6-mobile">
+          As of February 28, 2019, containerd is officially the fifth project to graduate from the Cloud Native Computing Foundation, behind
```
Comment (mikebrow) [inline layouts/partials/home/graduation.html]: /s/from/within/

**Comment by lucperkins** [comment]
@mikebrow Updated

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**